addons/mail/tests/common.py

```diff
@@ -581,8 +581,20 @@ def assertSentEmail(self, author, recipients, **values):
                     email_to_list.append(formataddr((email_to.name, email_to.email or 'False')))
                 else:
                     email_to_list.append(email_to)
-        expected['email_to'] = email_to_list
+        formatted_emails = []
+        for email_to in email_to_list:
+            emails_split = email_to.split(', ')
+            reconstructed = []
```

Comment:
@std-odoo done, but just letting you know, if in future we res_partner has two emails and I pass res_partners to the function it might cause a problems. scenario like I described [here](https://github.com/odoo/odoo/pull/158568#discussion_r1571994308)